src/features/MovieList/MovieTile/index.js

```diff
@@ -14,12 +14,16 @@ import {
     Votes,
     StyledStar,
     DescriptionContainer,
+    ImgWrapper,
 } from "./styled";
 
 export const MovieTile = ({ title, year, tags, rate, votes }) => {
     return (
         <Content>
-            <Img src={Poster} />
+            <ImgWrapper>
+                <Img src={Poster} />
+            </ImgWrapper>
+
```

Comment:
I can't change it now, I didn't remember what was there, and I will resolve the merge conflict if it will be necessary. And it is the result of my mistake when I made all changes on this branch rather than movie tile when I made changes with responsive